samcli/commands/init/__init__.py

```diff
@@ -231,6 +231,14 @@ def wrapped(*args, **kwargs):
     """ """,
     required=False,
 )
+@click.option(
+    "-t",
+    "--tracing",
+    default="disable",
+    type=click.Choice(["enable", "disable"]),
```

Comment:
Aren't we going to set the flag name as `--enable-tracing/--disable-tracing` which would be a boolean flag?